docs/oadp_monitoring.md

```diff
@@ -0,0 +1,235 @@
+<hr style="height:1px;border:none;color:#333;">
+<h1 align="center">OADP Observability</h1>
+<h2 align="center">Using OpenShift User Workload Monitoring for OADP</h2>
+
+## Preface
+
+The OpenShift Container Platform provides a [monitoring stack](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/index) that allows users and administrators to effectively monitor and manage their OpenShift clusters, as well as monitor and analyze the workload performance of user applications and services running on the clusters including receiving alerts when some events occurs.
+
+The OADP Operator leverages an OpenShift [User Workload Monitoring](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/enabling-monitoring-for-user-defined-projects) provided by the OpenShift Monitoring Stack for retrieving number of [metrics](#metrics) from the Velero service endpoint. The monitoring stack allows creating user-defined Alerting Rules or querying metrics using the OpenShift Metrics query front-end.
+
+With enabled User Workload Monitoring it is also possible to configure and use any Prometheus-compatible third-party UI, such as Grafana to visualize Velero metrics. Please note that the usage of third-party UIs falls outside the scope of this document.
+
+
+## OADP Monitoring Setup
+
+Monitoring [metrics](#metrics) requires enabling monitoring for the user-defined projects and creating ServiceMonitor resource to scrape those metrics from the already enabled OADP service endpoint that lives in the `openshift-adp` namespace.
+
+All of the configuration files from this document can be found in the [docs/examples/manifests/user_monitoring/](./examples/manifests/user_monitoring/) folder of this repository. They are named with the number prefix, which represents the order in which they should be applied.
+
+### Prerequisites
+* OADP operator, a credentials secret, and a DataProtectionApplication (DPA) CR are all created. Follow [these steps](/docs/install_olm.md) for installation instructions.
+* shell with the `oc` CLI command available and admin level access to the OpenShift cluster.
+
+
+### Enable and Configure User Workload Monitoring
+
+This paragraph will provide a short set of instructions how to enable user workload monitoring for an OADP project in the cluster. For comprehensive set of configuration options refer to the [enabling monitoring for user-defined projects](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/monitoring/enabling-monitoring-for-user-defined-projects#doc-wrapper) documentation.
+
+
+1. Create `1_enable_user_workload_monitoring.yaml` file that has enabled `enableUserWorkload`
```

Comment:
cant we just point to file location instead of telling the content again? (and if update something, we need to update 2 places... but if this goes to OCP docs later, better be like this)

Reply:
I expect this will be used in the docs, so that's why it's in both.